pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -135,29 +199,131 @@ def __init__(
             self,
             wires,
             *,
-            sync=True,
+            sync=False,
             c_dtype=np.complex128,
             shots=None,
             batch_obs: Union[bool, int] = False,
+            analytic=None,
```

Comment:
What does `analytic` control?

Reply:
Not sure, just a mirror from lightning's new implementation. I guess we can remove it, let me try later.

Reply:
removed